ca/ca_test.go

```diff
@@ -24,14 +24,14 @@ import (
 	"github.com/cloudflare/cfssl/helpers"
 	"github.com/cloudflare/cfssl/signer"
 	"github.com/cloudflare/cfssl/signer/local"
-	"github.com/google/certificate-transparency-go"
+	ct "github.com/google/certificate-transparency-go"
 	cttls "github.com/google/certificate-transparency-go/tls"
 	"github.com/jmhodges/clock"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/zmap/zlint/lints"
 	"golang.org/x/crypto/ocsp"
 
-	"github.com/letsencrypt/boulder/ca/config"
+	ca_config "github.com/letsencrypt/boulder/ca/config"
```

Comment:
What's up with this import alias diff? It doesn't seem like anything in this file is using `ca_config` (or the `ct` alias above)

Reply:
This is because there is a package name/import name mismatch. Each of these packages declare their names `package XXX` differently from the name of the import, which `goimports` doesn't like. If you look at the uses in the file currently they already used the name that is now being aliases (because of how golang treats these weird imports), this change is just bringing the name on the import line in line with the package name being used... if that makes sense?

Reply:
That makes sense, thanks :+1:

Reply:
i.e. if you import `github.com/letsencrypt/boulder/ca/config`, you can't actually refer to the package as `config` because it's package declaration is `package ca_config`, so any references have to be to `ca_config`.